module/map/map_fleet_preparation.py

```diff
@@ -142,17 +171,15 @@ def fleet_preparation(self):
         # Not using fleet 2.
         if not self.config.FLEET_2:
             if fleet_2.allow():
-                if fleet_2.in_use():
-                    fleet_2.clear()
+                fleet_2.clear()
             fleet_1.ensure_to_be(self.config.FLEET_1)
             self.map_fleet_checked = True
             return True
 
         # Using both fleets.
         # Force to set it again.
         # Fleets may reversed, because AL no longer treat the fleet with smaller index as first fleet
-        if fleet_2.in_use():
-            fleet_2.clear()
+        fleet_2.clear()

```

Comment:
Here should remain unchanged to avoid redundent clicks.
If no fleet is chosen, there's no need to click "CLEAR".

Reply:
in_use() is called at the beginning of clear() now. Nothing is actually clicked if no fleet is chosen, so here should be equivalent to the unchanged code I think.

Reply:
understood